core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/AutoTuner.scala

```diff
@@ -754,17 +754,32 @@ class AutoTuner(
       limitedLogicList: Option[Seq[String]] = Some(Seq("spark.sql.shuffle.partitions")),
       showOnlyUpdatedProps: Boolean = true):
       (Seq[RecommendedPropertyResult], Seq[RecommendedCommentResult]) = {
-    filterByUpdatedPropertiesEnabled = showOnlyUpdatedProps
-    limitedLogicList.foreach { limitedSeq =>
-      limitedSeq.foreach(_ => limitedLogicRecommendations.add(_))
-    }
-    skipList.foreach(skipSeq => skipSeq.foreach(_ => skippedRecommendations.add(_)))
-    if (processPropsAndCheck) {
-      initRecommendations()
-      calculateRecommendations()
+    val isOnGpu = {
+      val isPluginLoaded = getPropertyValue("spark.plugins") match {
+        case Some(f) => f.equals("com.nvidia.spark.SQLPlugin")
+        case None => false
+      }
+      val rapidsEnabled = getPropertyValue("spark.rapids.sql.enabled") match {
+        case Some(f) => f.toBoolean
+        case None => true
+      }
+      isPluginLoaded && rapidsEnabled
```

Comment:
If those two variables are not used later (aka, read them to add an appropriate comment), then we we don't need two variables to be calculated. It can be if condition.